.github/workflows/codeql.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         name: Update Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.22.9
+          go-version: 1.22.10
```

Comment:
Actually; this one may need quotes to prevent YAML from stripping the trailing zero (not sure if that still happens, but I recall I've been hit by that); let me do so